public/static/js/version_admin.js

```diff
@@ -76,8 +76,10 @@ mciModule.controller('AdminOptionsCtrl', [
                     $scope.closeAdminModal()
                     $rootScope.$broadcast("version_updated", data)
                     notifier.pushNotification(
-                      "Version " + (active ? "scheduled." : "unscheduled.") +
-                      (abort ? "\n In progress tasks will be aborted." : ""),
+                      "Version " + (active ? "scheduled." : "unscheduled") +
+                      (!active && $scope.version.Version.requester === "merge_test" ? " and removed from commit queue." : ".") +
```

Comment:
maybe the response from the server could indicate whether it's actually been removed? (maybe it's already not in the queue)